documentation/faq/60-env-vars.md

```diff
@@ -2,6 +2,6 @@
 question: How do I use environment variables?
 ---
 
-Vite uses [dotenv](https://github.com/motdotla/dotenv) to load environment variables from a file named `.env` or similar. Only environment variables prefixed with `VITE_` are exposed ([you can set `envPrefix` to change this](https://vitejs.dev/config/#envprefix)). For now, you would need to instantiate dotenv yourself if you want all environment variables exposed in `process.env['YOUR_ENV_VAR']`. [Vite 2.7 will expose all environment variables on the server-side](https://github.com/vitejs/vite/pull/5404).
+Vite uses [dotenv](https://github.com/motdotla/dotenv) to load environment variables from a file named `.env` or similar. Only environment variables prefixed with `VITE_` are exposed ([you can set `envPrefix` to change this](https://vitejs.dev/config/#envprefix)). For now, you would need to instantiate dotenv yourself if you want all environment variables exposed in `process.env.YOUR_ENV_VAR`.
```

Comment:
Should we update this to differentiate between client side and server side? I think the VITE_ prefix is only needed on the client side now?

Reply:
We want to be careful about "environment variables that are baked in at build time" versus "server-side environment variables that are accessed at runtime". My understanding is that the `VITE_` prefix is specifically about the former, and that it is a build-time replacement performed both in server code and client code. You'd need to handle it yourself (say with `$session`) if you wanted to expose runtime server-side environment variables to the client.